guides/v2.2/frontend-dev-guide/tools/using_grunt.md

```diff
@@ -41,7 +41,18 @@ Magento has built-in Grunt tasks configured, but there are still several steps y
 
 ## Grunt configuration file {#grunt_config}
 
-If installed as described above, Grunt will use the default configuration files located in the `dev/tools/grunt/configs/` directory. For example, the default configuration file for working with themes is `dev/tools/grunt/configs/themes.js`.
+If installed as described above, Grunt will use the default configuration files located in the `dev/tools/grunt/configs/` directory.  The default configuration file for working with themes is `dev/tools/grunt/configs/themes.js` and Difine your theme in `themes.js` For example,
+
+    Theme_Name: {
```

Comment:
This example code needs to be within:
```text 
<your example>
```

Reply:
the three backticks proceeding your example are not showing up in github, but they should be there along with three closing backticks after your example.

Reply:
Indenting a code sample causes it to render in a generic text code block automatically. It would be better to remove the excess spacing and add the following line before the code sample:

[backtick][backtick][backtick]javascript

where [backtick] = `

then add the following line after the code sample

[backtick][backtick][backtick]

Reply:
Thanks for clarifying @keharper!